ui-components/src/locales/ar.json

```diff
@@ -1,4 +1,7 @@
 {
+    "config": {
+        "routePrefix": "ar"
```

Comment:
So this routePrefix eventually becomes the "prefix" property on the language prop?

Reply:
Yes, this is what controls the routePrefix constant in the LanguageNav component: https://github.com/CityOfDetroit/bloom/blob/main/ui-components/src/navigation/LanguageNav.tsx#L21. I think LanguageNav should probably use the locale instead of a "translation item" though.